doc-src/ch-build_desc.rst

```diff
@@ -18,10 +18,19 @@ vagaries of making the supported builders work smoothly with Charliecloud and
 adds some conveniences (e.g., pass HTTP proxy environment variables to the
 build environment if the builder doesn't do this by default).
 
-Supported builders:
+Supported builders, unprivileged:
 
-  * :code:`ch-grow`: our internal builder; unprivileged
-  * :code:`docker`: Docker; privileged
+  * :code:`ch-grow`: our internal builder
+
+Supported builders, privileged:
+
+  * :code:`buildah-runc`: Buildah in "rootless" mode with setuid
+    helpers, using the default :code:`runc` for :code:`RUN` instructions
+
+  * :code:`buildah-setuid`: Buildah in "rootless" mode with setuid helpers,
+    using :code:`ch-run` (via :code:`ch-run-oci`) for :code:`RUN` instructions
+
```

Comment:
A little confused by this section. So buildah-setuid uses the oci shim, but still uses new[u,g]idmap?

Reply:
Yes, that's correct.

Reply:
Ah! So fully unprivileged buildah isn't supported with this patch since we still have logistical issues to work out?